client/src/components/_common/TextCopyField/TextCopyField.module.scss

```diff
@@ -0,0 +1,20 @@
+.root {
+  display: flex;
+}
+
+.input {
+  composes: form-control from '../../../styles/components/bootstrap.form.css';
+}
+
+.copy-button {
+  composes: c-button--secondary from '../../../styles/components/c-button.css';
+}
+
+.button__icon {
+  composes: c-button__icon--before from '../../../styles/components/c-button.css';
+}
```

Comment:
__Note to Reviewers__: This unfortunately-verbose method is, currently, the most future-proof way to style buttons; [FP-545](https://jira.tacc.utexas.edu/browse/FP-545) will provide an in-house `_common` button component that does this for y'all. I have been delayed in its implementation.